src/ui/page/home/home.tsx

```diff
@@ -5,16 +5,19 @@ import { Explore } from './explore/explore'
 import './home.scss'
 import './i18n/index'
 
-export const Home: FC = () => (
-  <div className="okp4-dataverse-portal-home-page-main">
-    <div className="okp4-dataverse-portal-home-page-block-container community">
-      <Community />
+export const Home: FC = () => {
+  throw new Error()
```

Comment:
Do we keep it for testing purpose?

Reply:
Ooops, completely forgot to remove this🤦🏼‍♀️